docs/new-audits.md

```diff
@@ -8,13 +8,33 @@ Lighthouse audits that surface in the report should be:
 - Contribute significantly towards making the mobile web experience better for end users. 
 - Not have a significant impact on our runtime performance or bundle size. 
 - Something that is new, and not something that is already measured by existing audits. 
-- Important for our strategic goals as a product.
-- Measurable (especially for performance audits) or have clear pass/fail states. 
+- Measurable (especially for performance audits) or have clear pass/fail states.
+- Actionable - when failing, specific advice should be given. If the failure can be tied to a specific resource (a DOM element, script, line of code), use the [appropriate detail type](https://github.com/GoogleChrome/lighthouse/blob/master/types/audit-details.d.ts). If multiple failures can occur for a page, return a table.
 - Not use 3rd party APIs for completing the audit check. 
 
+## Actionable
```

Comment:
actuallly... lets put this actionability section in the plugins.md.  that has a bigger audience than this doc.  we can of course add a link from new-audits to the section.

Reply:
I already added a link from the plugin doc -> new audit doc. I don't think this file should exist if we put important new-audit advice elsewhere. weird bifurcation imo

Reply:
we have two docs for two "conceivable" audiences:

* audience who wants to build a plugin full of their own audits
* audience who wants to propose a new audit to core (but doesnt want to implement as a plugin)

The actionability stuff is relevant to both of them.

I'm just saying the 1st audience is bigger than the second.  Really I think this actionability stuff is a Must-read for both audiences but i'd rather the plugin folks see it.  The risk is greater for them; they could end up making a useless audit. Whereas the proposers just need us to provide feedback to improve their proposal.

The new-audit doc still stands on its own afterwards.

i'm happy to do it if you'd rather not spend the time. (though i would like to convince you its better this way. :)